src/libraries/System.Private.Xml/src/System/Xml/XmlConvert.cs

```diff
@@ -761,28 +754,35 @@ public static string ToString(Guid value)
 
         public static bool ToBoolean(string s)
         {
-            s = TrimString(s);
```

Comment:
Is it possible to remove the `TrimString` helper method entirely?

Reply:
There are 5 references left :
2 in XmlConvert and used to parse to Uri, but I didn't found method in Uri accepting Span - please share if there is.
1 in XmlValueConverter :` Convert.FromBase64String(XmlConvert.TrimString(value));`. Here I found TryFromBase64Chars with span, but it need destination with correct size. There is no sample in whole runtime repo :) .
1 in XmlWellFormedWriter - but it expects string
1 more there - which I will remove

@eiriktsarpalis